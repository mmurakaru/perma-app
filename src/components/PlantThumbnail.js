import styled from 'styled-components/macro'

export default function PlantThumbnail({ imgUrl, name }) {
    return (
        <Thumbnail>
            <img src={imgUrl} alt="" />
            <h2>{name}</h2>
        </Thumbnail>
    )
}

const Thumbnail = styled.div`
    img {
        height: 100px;
        width: 146px;
        border-radius: 5px;
    }
    h2 {
        font-weight: 500;
        font-size: 0.8rem;
    }
`
