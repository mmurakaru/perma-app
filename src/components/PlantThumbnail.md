PlantThumbnail example:

```js
<PlantThumbnail imgUrl="https://source.unsplash.com/random" name="Strawberry" />
```
