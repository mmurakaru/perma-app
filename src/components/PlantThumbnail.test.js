import PlantThumbnail from './PlantThumbnail'
import { render } from '@testing-library/react'

describe('PlantThumbnail', () => {
    it('renders correctly', () => {
        const { container } = render(
            <PlantThumbnail
                name="Strawberyy"
                imgUrl="https://source.unsplash.com/random"
            />
        )
        expect(container.firstChild).toMatchSnapshot()
    })
})
