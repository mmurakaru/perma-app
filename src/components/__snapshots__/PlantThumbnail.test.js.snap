// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlantThumbnail renders correctly 1`] = `
<div
  class="PlantThumbnail__Thumbnail-qnblyr-0 fSfroD"
>
  <img
    alt=""
    src="https://source.unsplash.com/random"
  />
  <h2>
    Strawberry
  </h2>
</div>
`;
